lib/span.js

```diff
@@ -241,6 +241,7 @@ Span.prototype.runAsync = function (fn) {
       if (span.topSpan && span.doMetrics) {
         const txname = span.getTransactionName()
         const et = (new Date().getTime() - startTime) * 1000
+        spansTopSpanTime += et;
```

Comment:
is this milliseconds or microseconds? I don't know what it should be?

Reply:
it's microseconds. i guess in the node world it's clear that (new Date().getTime()) returns milliseconds so * 1000 is microseconds. but a comment wouldn't hurt :-)